alchemy/src/cloudflare/bundle/bundle-worker-dev.ts

```diff
@@ -0,0 +1,106 @@
+import type esbuild from "esbuild";
+import type { Bindings } from "../bindings.ts";
+import type { WorkerProps } from "../worker.ts";
+
+interface DevWorkerContext {
+  context: esbuild.BuildContext;
+  dispose: () => Promise<void>;
+}
+
+const activeContexts = new Map<string, DevWorkerContext>();
```

Comment:
Place this on globalThis to ensure global singleton. Recently ran into problems with monorepo